added an installation section to CMakeLists.txt

marker68 opened the pull request:
For the use of -DCMAKE_INSTALL_PREFIX

maxhutch committed:
2.0.2 (May 2, 2016)

 - Adds MAGMA_NO_V1 option to disable MAGMA v1.x compatability.
 - Updates the testers to use MAGMA v2.0.
 - Fixes use of NULL stream when using MAGMA v1.x interface.

marker68 committed:
added an installation section to CMakeLists.txt

marker68 committed:
fixed a small bug

marker68 committed:
fixed a bug on lib installation

maxhutch commented:
Hi @marker68 ,

I don't actually develop Magma; I just mirror it here for developers that prefer git-style distribution or use git submodules.  If you want to contribute changes upstream, then you might want to post on the [Magma User Forum](http://icl.cs.utk.edu/magma/forum/viewforum.php?f=2).  I'll leave this PR open so you can show them [the diff](https://github.com/maxhutch/magma/pull/1/files), if you want.  I don't know if they even know this mirror is here.

If, for whatever reason, they aren't able to accommodate your change, then I can make a branch for user modifications and you could put your changes there.  They look helpful!  We could get travis running as well to check for regressions.

Let me know how it goes,
Max

marker68 commented:
Thank you for your reply. Anyway, I will continue to update in a separate branch in my forked repo.

maxhutch committed:
MAGMA 2.1 release 2016-08-30

MAGMA 2.1 for CUDA is now available. New features and updates include:
 * Variable size batched routines (gemm, gemv, syrk, syr2k).
 * Improved SVD performance for tall (m >> n) or wide (m << n) matrices.
 * Preconditioned QMR.
 * Expanded doxygen documentation.
 * For MAGMA v1 compatability, initializes default queue for each GPU on
   first use, instead of in magma_init.

Please take [this survey](https://www.surveymonkey.com/r/2016DenseLinearAlgebra)
to help improve MAGMA, LAPACK, and other dense
linear algebra libraries. We estimate that it should take 10 minutes to
fill it out.  Thank you very much.

marker68 committed:
Merge branch 'features/installation' of https://github.com/marker68/magma into features/installation

* 'features/installation' of https://github.com/marker68/magma:
  fixed a bug on lib installation
  fixed a small bug
  added an installation section to CMakeLists.txt
